arviz/tests/base_tests/test_labels.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from ...labels import (
+    BaseLabeller,
+    DimCoordLabeller,
+    DimIdxLabeller,
+    IdxLabeller,
+    MapLabeller,
+    NoModelLabeller,
+    NoVarLabeller,
+    mix_labellers,
+)
+
+
+class Data:
+    def __init__(self):
+        self.sel = {
+            "instrument": "a",
+            "experiment": 3,
+        }
+        self.isel = {
+            "instrument": 0,
+            "experiment": 4,
+        }
+
+
+@pytest.fixture
+def multidim_sels():
+    return Data()
+
+
+class Labellers:
+    def __init__(self):
+        self.labellers = {
+            "BaseLabeller": BaseLabeller(),
+            "DimCoordLabeller": DimCoordLabeller(),
+            "IdxLabeller": IdxLabeller(),
+            "DimIdxLabeller": DimIdxLabeller(),
+            "MapLabeller": MapLabeller(),
+            "NoVarLabeller": NoVarLabeller(),
+            "NoModelLabeller": NoModelLabeller(),
+        }
+
+
+@pytest.fixture
+def labellers():
+    return Labellers()
+
+
+@pytest.mark.parametrize(
+    "args",
+    [
+        ("BaseLabeller", "theta\na, 3"),
+    ],
+)
+class TestLabellers:
+    def test_make_label_vert(self, args, multidim_sels, labellers):
+        labeller_arg = labellers.labellers[args[0]]
+        label = labeller_arg.make_label_vert("theta", multidim_sels.sel, multidim_sels.isel)
+        assert label == args[1]
```

Comment:
very minor nit, for readability I (aka present me who seems a bit wiser than old me) would probably unpack the `args` into something like `name, expected_label = args` and then use those inside the test instead of `args[0]`.